docs/neo-n3/faq.md

```diff
@@ -0,0 +1,36 @@
+# Frequently Asked Questions
+
+This page contains questions that we've been asked in the past and might arise during your development process.
+
+### General Questions
+
+##### Why do I need docker installed for running the test framework?
+
+The test framework runs a neo-node instance inside of a docker container. This allows you to test your smart contract
+and invocations with a running Neo N3 blockchain.
+
+##### How can I make use of Neo's fine granularity for transaction signatures?
+
+For this, we suggest you to read through
+[this Medium article](https://neospcc.medium.com/thou-shalt-check-their-witnesses-485d2bf8375d) about the mechanisms
+around signers, scopes and witnesses.
+
+### SDK - dApp Development
+
+##### What RPC is used to relay a transaction that I will pay for, and which ones are just for reading?
+
+The RPC `sendrawtransaction` is used to relay transactions that have been signed and will persist on-chain (i.e., it
+costs GAS). The RPCs `invokefunction` and `invokescript` are read-only and can be used to "mock" invocations to figure
+out what would happen if this function or script was invoked/executed in a signed transaction.
+
+##### How does the node validate a witness check when I only use read invocations?
+
+When invoking methodds with `invokefunction` or `invokescript` RPCs, calls to `checkWitness` return true as long as the
+correct signers (with the correct scope) are attached in the request.
```

Comment:
Maybe append: "But the actual signature is not checked"

Reply:
There's no signature anyways. I think adding this could confuse the reader. I'll not append this as discussed with you in person.